src/components/Button/PrimaryButton.tsx

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components';
+
+export interface ButtonProps {
+  type?: 'submit' | 'button';
+  children: string;
+  onClick: () => void;
+  state?: boolean;
+  size: 'large' | 'medium' | 'small';
+}
+
+const PrimaryButton = ({
+  type,
+  children,
+  onClick,
+  state,
+  size,
+}: ButtonProps) => {
+  return (
+    <Button type={type} onClick={onClick} $isActive={state} size={size}>
+      {children}
+    </Button>
+  );
+};
+
+export default PrimaryButton;
+
+const Button = styled.button<{
+  $isActive?: boolean;
+  size: 'large' | 'medium' | 'small';
```

Comment:
이런 union 타입의 경우에는 추후에 변동 가능성을 고려하면 별도로 만드는 것보단 위에서 받아오는게 더 유지보수성이 좋을듯!

```suggestion
  size: ButtonProps['size'];
```